cfy_manager/components/mgmtworker/mgmtworker.py

```diff
@@ -85,6 +85,16 @@ def _deploy_hooks_config(self):
         file_name = 'hooks.conf'
         config_dir = join(HOME_DIR, 'config')
         hooks_config_dst = join(config_dir, file_name)
+
+        # If the hooks config file already exists, do nothing. This file
+        # can be altered by users, so we shouldn't overwrite it once present.
+        # Can't use os.path.exists because of the file's permissions
```

Comment:
What are the permissions that make it not work with os.path.exists, but still work with ls?

Reply:
Just the fact that it is chowned by cfyuser